packages/react/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -62,8 +62,15 @@ const getFocusedStyles = ({
         }
       : {
           color,
-          background: v.focusedBackgroundColor,
+          background: v.hoverBackgroundColor,
         }),
+
+    ...(vertical && isFromKeyboard && !pointing && !primary
```

Comment:
could this just work for all non iconOnly and non pointing menus, meaning the default horizontal menu? it's not worth a lot of work to get it working because we don't have that scenario in Teams to my understanding, but if it just works or is close that would be nice.

Reply:
To be honest it doesn't look that great on other use cases because of the borders/underlines there already.

It's worth coming back to this when we've an understanding where we go with the menu variations altogether. The component styles should be rewritten from ground up once we've mapped our needs.